stl/inc/bitset

```diff
@@ -406,12 +390,12 @@ public:
 
         constexpr bool _No_padding = _Bits % _Bitsperword == 0;
         for (size_t _Wpos = 0; _Wpos < _Words + _No_padding; ++_Wpos) {
-            if (_Array[_Wpos] != ~static_cast<_Ty>(0)) {
+            if (_Array[_Wpos] != ~_Ty{0}) {

```

Comment:
Unrelated change for consistency. In other similar places there is `{ }`. The inconsistency was introduced by me a year ago.